airsonic-main/src/main/java/org/airsonic/player/domain/entity/Share.java

```diff
@@ -58,7 +51,7 @@ public class Share {
     @Column(name = "visit_count")
     private int visitCount;
 
-    @OneToMany(mappedBy = "share", cascade = { CascadeType.MERGE, CascadeType.REMOVE, CascadeType.PERSIST })
+    @OneToMany(mappedBy = "share", fetch = FetchType.EAGER, cascade = { CascadeType.MERGE, CascadeType.REMOVE, CascadeType.PERSIST })
```

Comment:
The share's files weren't accessible in the share settings page, so I added fetch=EAGER to ensure they were available on that page.